client/app/telecom/telephony/alias/special/fees/telecom-telephony-alias-special-fees.html

```diff
@@ -0,0 +1,147 @@
+                                    data-ng-disabled="!AliasSpecialFeesCtrl.fees.length"
+                                    data-ng-click="AliasSpecialFeesCtrl.orderBy('price')">
+                                <span data-translate="telephony_alias_special_fees_price_filter"></span>
+                                <span>
+                                        <i class="oui-icon oui-icon-arrow-down"
+                                           data-ng-show="AliasSpecialFeesCtrl.state.orderBy === 'price' && AliasSpecialFeesCtrl.state.orderDesc">
+                                        </i>
+                                        <i class="oui-icon oui-icon-arrow-up"
+                                           data-ng-show="AliasSpecialFeesCtrl.state.orderBy === 'price' && !AliasSpecialFeesCtrl.state.orderDesc">
+                                        </i>
+                                    </span>
+                            </button>
+                        </th>
+                    </tr>
+                    </thead>
+                    <tbody>
+                    <tr data-ng-repeat="fee in AliasSpecialFeesCtrl.fees | orderBy:AliasSpecialFeesCtrl.state.orderBy:AliasSpecialFeesCtrl.state.orderDesc track by $index">
+                        <th style="width: 45px;">&nbsp;</th>
+                        <td data-ng-bind="fee.creationDatetime | date:'dd-MM-yyyy hh:mm:ss'"></td>
+                        <td data-ng-bind="fee.calling"></td>
+                        <td data-ng-bind="fee.duration"></td>
+                        <td data-ng-bind="fee.operator"></td>
+                        <td data-ng-bind="fee.price"></td>
+                    </tr>
+                    </tbody>
+                    <tfoot>
+                    <tr>
+                        <th class="font-weight-bold"
+                            scope="row"
+                            colspan="2"
+                            data-translate="telephony_alias_special_fees_total">
+                        </th>
+                        <td class="font-weight-bold"
+                            data-ng-bind="'telephony_alias_special_fees_calls' | translate: {calls: AliasSpecialFeesCtrl.totalRepayment.calls || 0}"></td>
+                        <td colspan="2"
+                            class="font-weight-bold"
+                            data-ng-bind="'telephony_alias_special_fees_duration' | translate: {duration: AliasSpecialFeesCtrl.totalRepayment.duration || 0}"></td>
+                        <td class="font-weight-bold"
+                            data-ng-bind="'telephony_alias_special_fees_price' | translate: {price: AliasSpecialFeesCtrl.totalRepayment.price || 0}"></td>
+                    </tr>
+                    </tfoot>
+                </table>
+            </div>
+            <div class="text-center"
+                 data-ng-if="AliasSpecialFeesCtrl.isLoading">
```

Comment:
During the loading state you'll have:

```html
<div class="row">
    <div class="text-center"
         data-ng-if="AliasSpecialFeesCtrl.isLoading">
         <oui-spinner></oui-spinner>
    </div>
</div>
```

It result with a Bootlint error E013 according to doc: https://github.com/twbs/bootlint/wiki/E013